python/ray/experimental/client/client_app.py

```diff
@@ -3,6 +3,27 @@
 ray.connect("localhost:50051")
 
 
+@ray.remote
+class HelloActor:
+    def __init__(self):
+        self.count = 0
+
+    def say_hello(self, whom):
+        self.count += 1
+        return ("Hello " + whom, self.count)
+
+
+actor = HelloActor.remote()
+s, count = ray.get(actor.say_hello.remote("you"))
+print(s, count)
+assert s == "hello you"
+assert count == 1
+s, count = ray.get(actor.say_hello.remote("world"))
+print(s, count)
+assert s == "hello world"
+assert count == 2
```

Comment:
One thing I noticed is that passing actor handles crashes trying to serialize the handle:

```
@ray.remote
def f(handle):
    return handle.say_hello.remote("Bob")

ray.get(f.remote(actor))
```